prod.Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM openjdk:11.0.13-jdk-bullseye as build-stage
+FROM openjdk:17.0.2-jdk-bullseye as build-stage
 
 RUN mkdir /src
 COPY . /src
 WORKDIR /src
 RUN chmod +x mvnw
 RUN ./mvnw clean package -Dmaven.test.skip=true
 
-FROM openjdk:11.0.13-jre-bullseye
+FROM openjdk:17.0.2-jdk-bullseye
```

Comment:
This change is going to need extensive manual testing before approving, as we are upgrading through 6 major JDK/JRE versions in the production image, which is very difficult to test locally.